packages/ui-components/style/collapse.css

```
@@ -7,19 +7,32 @@
   display: flex;
   flex-direction: column;
   align-items: stretch;
-  border-top: 1px solid var(--jp-border-color2);
-  border-bottom: 1px solid var(--jp-border-color2);
 }
 
 .jp-Collapse-header {
   padding: 1px 12px;
-  color: var(--jp-ui-font-color1);
   background-color: var(--jp-layout-color1);
-  font-size: var(--jp-ui-font-size2);
+  border-bottom: solid var(--jp-border-width) var(--jp-border-color2);
+  color: var(--jp-ui-font-color1);
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  font-size: var(--jp-ui-font-size0);
+  font-weight: 600;
+  text-transform: uppercase;
+}
```

```suggestion
  user-select: none;
}
```

One difference I noted when testing on binder is that when clicking on headers in property inspector the titles sometimes get selected whereas in extension manager or debugger this does not happen thanks to `user-select: none`.